.yamato/config/_latest_editor_versions.metafile

```diff
@@ -0,0 +1,57 @@
+# WARNING: This file is automatically generated.
+# To add new Unity Editor tracks, the script needs to be updated.
+
+editor_version_names:
```

Comment:
About the naming: Would it make sense to follow the "convention" of the Unity-QA-MultiTool software? 
- `trunk_latest_internal` would become `trunk_promoted`
- `trunk_latest_public` would become `trunk_published`
- I don't know if there's an equivalent for the staging one

Reply:
Good one! Right now it is kept consistent with the dots (the editor pinning is based on their solution), so this is a matter of preference. But is internal promoted and public published, as they seem to go opposite with packages convention?